RatRazor/Pages/Overlay/Index.razor

```diff
@@ -16,19 +17,27 @@
         {
             @if (itemScan is ItemIconScan iconScan)
             {
-                <div style="font-size: smaller; position: absolute; left: @(iconScan.GetToolTipPosition().X - VirtualScreenOffset.XOffset)px; top: @(iconScan.GetToolTipPosition().Y - VirtualScreenOffset.YOffset)px; width: @(iconScan.Icon.ItemSize.X)px; height: @(iconScan.Icon.ItemSize.Y)px; ">
+                <div style="font-size: smaller; position: absolute; left: @(GetItemOverlayPosition(itemScan).X + 3)px; top: @(GetItemOverlayPosition(itemScan).Y + 3)px; width: @(iconScan.Icon.ItemSize.X)px; height: @(iconScan.Icon.ItemSize.Y)px; ">
```

Comment:
This is now calculated relative to the overlay's position (which fits the game window)